pytype/pyc/opcodes.py

```diff
@@ -440,8 +440,8 @@ class SETUP_ANNOTATIONS(Opcode):
   __slots__ = ()
 
 
-class YIELD_VALUE(Opcode):
-  _FLAGS = HAS_JUNKNOWN
+class YIELD_VALUE(OpcodeWithArg):
+  _FLAGS = HAS_ARGUMENT
```

Comment:
How do we make this backwards compatible? Out of the gate this breaks in 3.11, because it expects YIELD_VALUE to have no arguments.

We're getting the following error:

```
Traceback (most recent call last):
  File "/tmp/pytype/blocks/blocks_test.py", line 88, in test_yield
    b0, b1 = ordered_code.order
    ^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)
```

@martindemello, Thomas said you might have time and motivation to help a little. Do you have an idea how to make this change backwards compatible? Can we somehow have 2 different YIELD_VALUE versions in the code or is that not possible, yet? If you don't have time or don't want to right now, please just ignore this. We're going to figure this out somehow. 🙂

Reply:
I mean, a nested class works. It feels really ugly. But ... maybe that's fine? It keeps __class__.__name__ the same, which might be important?

Reply:
that's interesting, I don't think we have had an opcode go from no arg to arg before! keeping class.name the same is indeed important, if a nested class works that might be the way to go until you can think of something better. you can define the class within an "if sys.version" block but that would lose the ability to have host and target version different (we might have already given up on that? I can't remember, we certainly discussed it).

you will definitely also need to version guard the tests, there are examples of that elsewhere in the code.

Reply:
I took a closer look at the diffs and for_version does feel like a good solution, it's nicely contained in a small part of the code

Reply:
Thanks a lot, Martin. Thanks also for the hint about about the tests. I didn't get that far, yet. I'll have a look at that in the next days.